### ISSUE TITLE
Validate JSDoc argument number

### ISSUE THREAD

**Issue by ntadej** [angelozerr/tern-lint#55]
Validate if argument is required or not in JSDoc.
For example:
```javascript
    /**
     * Return string based on error code
     * @param {number} id - error id
     * @param {string=} message - error message
     * @return {string} localized error string
     */
    error(id, message) { ... }
```
Here, `error()` should be marked as invalid, as `id` is required, whereas `error(5)` would be valid.

**Comment by angelozerr** [comment]
I agree that it should be cool if we have this feature which looks like https://github.com/angelozerr/tern-lint/issues/32#issuecomment-113758167

I must study if tern can manage very well optional parameter to avoid having a lot of errors.
